index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <link rel="stylesheet" href="style.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>my-portfolio</title>
+</head>
+<body>
+    <div class="container">
+        <header class="navbar">
+            <nav>
+                <a href="#">
+                    <img class="my-logo" src="images/My Logo.png" alt="logo">
+                </a>
+                <a href="#">
+                    <img class="menu" src="images/union.png" alt="union">
+                </a>
```

Comment:
- [ ] As suggested by the previous reviewer please fix the menu button, before you merge.
![port8](https://user-images.githubusercontent.com/55635977/129346978-654e3129-39a7-4afb-9172-311a06c0f90e.png)